charts/stable/cops/values.yaml

```diff
@@ -21,8 +19,8 @@ service:
   main:
     ports:
       main:
-        port: 80
-        protocol: TCP
+        port: 12480
```

Comment:
the next port should have been `10609` :P 
https://truecharts.org/charts/default_port_list#ports-that-are-blocked-in-major-web-browsers

Reply:
lets make it _more_ standard.

```
image:
  pullPolicy: IfNotPresent
  repository: tccr.io/truecharts/cops
  tag: latest@sha256:f356015ceef5155bdab78191c803f03affc80392dd9e24c795395dd4543b947c

podSecurityContext:
  runAsGroup: 0
  runAsUser: 0

securityContext:
  readOnlyRootFilesystem: false
  runAsNonRoot: false

service:
  main:
    ports:
      main:
        port: 10609
        protocol: HTTP
        targetPort: 80

persistence:
  config:
    enabled: true
    mountPath: /config
  books:
    enabled: true
    mountPath: /books

portal:
  enabled: true
```

Reply:
Good call, let me clean up more